.github/workflows/test-and-publish-nightly.yml

```diff
@@ -148,6 +148,27 @@ jobs:
             java-version: 11
             java-distribution: corretto
             image: public.ecr.aws/async-profiler/asprof-builder-arm:latest
+          - runson:
+              display: linux-arm64
+              # There is no "latest" tag available (yet) as ARM runners are still in public preview
```

Comment:
I think one of these comments is enough, especially because it does not seem to be a real problem for us.